scripts/deploy_docker.sh

```diff
@@ -7,12 +7,14 @@ export DOCKER_BUILDKIT=1
 if [ "$1" = "develop" -o "$1" = "master" ]; then
     # If image does not exist, don't use cache
     docker pull gnosispm/$DOCKERHUB_PROJECT:$1 && \
-    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:$1 || \
-    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile .
+    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:$1 --build-arg BUILDKIT_INLINE_CACHE=1 || \
+    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --build-arg BUILDKIT_INLINE_CACHE=1
 else
+    # Building tag version from staging image (vX.X.X)
     docker pull gnosispm/$DOCKERHUB_PROJECT:staging && \
-    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:staging || \
-    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile .
+    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:staging --build-arg BUILDKIT_INLINE_CACHE=1 || \
+    docker build -t $DOCKERHUB_PROJECT -f docker/web/Dockerfile . --build-arg BUILDKIT_INLINE_CACHE=1
+    docker tag $DOCKERHUB_PROJECT gnosispm/$DOCKERHUB_PROJECT:latest
```

Comment:
This `tag` command won't run for either `master` nor `develop` . Is that ok?

Reply:
That's alright. I only want to mark as `latest` the last stable (similar to what a lot of docker images do). That way people can pin `latest` in their deployments knowing that they are running an updated version (even if not recommendable to blindly update)